.pytool/Plugin/ImageValidation/ReadMe.md

```diff
@@ -7,13 +7,32 @@ file path is provided via the command line as `PE_VALIDATION_PATH=<PATH>` or
 can be configured in the the PlatformBuild.py within the `SetPlatformEnv()`
 method using 
 `self.env.SetValue("PE_VALIDATION_PATH", <PATH>, "Platform Hardcoded")`. A 
-profile is equivalent to the file types defined in the platform's fdf. All
-profiles must be defined, forcing the developer to acklowedge each, however
+profile is equivalent to the binary's MODULE_TYPE as defined in it's inf. All
+profiles must be defined, forcing the developer to acknowledge each, however
 requirements for each profile do not need to be specified... If one or more
 requirement does not exist, the "DEFAULT" profile requirements will be used.
 The developer can have default requirements via the "DEFAULT" profile then
 override those requirements in other profiles. An example of a full config
-file can be seen at the bootom of the readme.
+file can be seen at the bottom of the readme.
+
+A default set of requirements are provided by the tool that apply to all
+X64 and AARCH64 binaries. These requirements are:
+
+1. IMAGE_BASE = 0x0
+2. 4k Section Alignment (X64, AARCH64), 64k Section Alignment (AARCH64-DXE_RUNTIME_DRIVER)
+3. No Section can be both Write and Execute.
```

Comment:
Can we add a justification for why these are the default reqs? Something along the lines of if image base is non-zero, some PE loaders (including edk2's) will attempt to load the image at that address, which is generally not needed in a UEFI env, 4k section alignment is required in DXE for memory protections on images, and 64k aarch64 runtime driver alignment is required by the UEFI spec, w/x is against mem prot

Reply:
Updated documentation to include this. Good idea.